build.gradle

```diff
@@ -87,6 +87,9 @@ test {
     testLogging {
         exceptionFormat = 'full'
     }
+    afterTest { desc, result ->
+        println "Executing test ${desc.name} [${desc.className}] with result: ${result.resultType}"
```

Comment:
For posterity: This is here so that I can check my test(s) actually ran. It emits the test name and result into the build log. Otherwise the only time I am sure my test ran is by deliberately making it fail.

I know that I could run gradle in INFO and DEBUG mode which would also display the test output, but it also fills the console with additional buildtime fluff.